pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-ingestion-common/pom.xml

```diff
@@ -33,6 +33,12 @@
   <artifactId>pinot-ingestion-common</artifactId>
   <name>Pinot Ingestion Common</name>
   <url>https://pinot.apache.org/</url>
+  <repositories>
```

Comment:
why we need it here?

Reply:
It was failing without it during compilation.